tutorials/thinking.md

```diff
@@ -82,7 +90,24 @@ var todo
 <template v-slot:rust>
 
 ```rust
-let todo
+use libipld::{ipld, Ipld};
+
+fn index(person: Ipld, planet: &mut Vec<Ipld>) {
+    planet.push(person)
+}
+
+fn main() {
```

Comment:
You can probably pull this in rather than nesting in a main() function. The only reason I did this in the JS example is because it uses `await` so it needs to be in an async function until top level await lands in the language.